kai/service/llm_interfacing/model_provider.py

```diff
@@ -137,3 +153,40 @@ def __init__(self, config: KaiConfigModels):
             ]
         else:
             self.llama_header = config.llama_header
+
+
+class Wrapper:
```

Comment:
where is this being used?

Reply:
I wanted to see how the ModelProvider's LLM was used during debugging. I'll move this to the test module.